SwiftUIDZ/SwiftUIDZ/CarsView.swift

```diff
@@ -0,0 +1,235 @@
+                    .foregroundColor(.white)
+                createDistanceSlider()
+                HStack(spacing: Constants.spacing) {
+                    Text(toggleIsOn ? "\(Constants.totalText) \(rents[segmentIndexRentalPeriod].price + 100) $" :
+                            "\(Constants.totalText) \(rents[segmentIndexRentalPeriod].price) $")
+                    .font(Font.custom(Constants.fontName, size: Constants.fontSizeRegular).weight(.bold))
+                    createRentButton()
+                    createShareButton()
+                }
+                .padding()
+                .navigationBarHidden(true)
+            }
+        }
+    }
+    
+    // MARK: - Private Methods
+    
+    private func createCarParameter(imageName: String, parameterText: String) -> some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: Constants.cornerRadius)
+            VStack {
+                Image(imageName)
+                    .resizable()
+                    .renderingMode(.template)
+                    .foregroundColor(.purple)
+                    .frame(width: Constants.parameterIconSize, height: Constants.parameterIconSize)
+                Text(parameterText)
+                    .foregroundColor(.white)
+                    .font(Font.custom(Constants.fontName, size: Constants.fontSizeSmall).weight(.bold))
+            }
+        }
+    }
+    
+    private func createCarNameSegment() -> some View {
+        Picker(selection: Binding(get: {
+            self.segmentIndex
+        }, set: { newValue in
+            self.segmentIndex = newValue
+            self.offset = -10
+            self.moveBack()
+        })) {
+            ForEach(0..<cars.count, id: \.self) { index in
+                Text(cars[index].name)
+            }
+        } label: {
+            Text("")
+        }
+        .pickerStyle(.segmented)
+        .padding(.horizontal)
+    }
+    
+    private func createForm() -> some View {
+        Form {
+            Section {
+                Picker(selection: $segmentIndexRentalPeriod) {
+                    ForEach(0..<rents.count, id: \.self) { index in
+                        Text(rents[index].period)
+                    }
+                } label: {
+                    Text(Constants.rentalText)
+                        .font(Font.custom(Constants.fontName, size: Constants.fontSizeSmall).weight(.bold))
+                }
+                Toggle(Constants.fillText, isOn: $toggleIsOn)
+                    .tint(.purple)
+                    .font(Font.custom(Constants.fontName, size: Constants.fontSizeSmall).weight(.bold))
+            }
+        }
+    }
+    
+    private func createDistanceSlider() -> some View {
+        HStack {
+            Slider(value: Binding(get: {
+                progress
+            }, set: { newValue in
+                progress = newValue
+            }), in: 0...50)
```

Comment:
```suggestion
            }), in: 0...50)
```
Можно в константы весь диапазон